jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/UpdateCheckerTest.java

```diff
@@ -74,16 +74,16 @@ public void testPerformUpdateCheck_newVersionFound() throws IOException {
     Optional<String> message =
         UpdateChecker.performUpdateCheck(
             configDir, "1.0.2", testWebServer.getEndpoint(), "tool name", ignored -> {});
-    Assert.assertTrue(testWebServer.getInputRead().contains("User-Agent: jib 1.0.2 tool name"));
-    Assert.assertTrue(message.isPresent());
-    Assert.assertEquals(
-        "A new version of Jib (2.0.0) is available (currently using 1.0.2). Update your build "
-            + "configuration to use the latest features and fixes!",
-        message.get());
+
+    assertThat(testWebServer.getInputRead()).contains("User-Agent: jib 1.0.2 tool name");
+    assertThat(message)
+        .hasValue(
+            "A new version of Jib (2.0.0) is available (currently using 1.0.2). Update your build "
+                + "configuration to use the latest features and fixes!");
     String modifiedTime =
         new String(
             Files.readAllBytes(configDir.resolve("lastUpdateCheck")), StandardCharsets.UTF_8);
-    Assert.assertTrue(Instant.parse(modifiedTime).isAfter(before));
+    assertThat(Instant.parse(modifiedTime).isAfter(before)).isTrue();
```

Comment:
Truth has `assertThat(Instant)`, so you can probably do like `assertThat(modifiedTime).isAfter(before)` or something like that.